src/components/Modals/NewCourse/CourseSelector.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div class="autocomplete">
+    <input
+      v-bind:class="['search-box', searchBoxClassName]"
+      ref="dropdownInput"
+      v-model="searchText"
+      :placeholder="placeholder"
+      @keyup.esc="onEscape"
+      @keyup.enter="onEnter"
+      @keyup.up="currentFocus -= 1"
+      @keyup.down="currentFocus += 1"
```

Comment:
I think the way we handle incrementing/decrementing `currentFocus` might need to be adjusted so that we have the looping mechanism that we see on the right panel when using the up and down arrow keys. Maybe once `currentFocus` is at the max intended value (I think maybe 9 since we show 10 results?), we don't keep incrementing. I think `addActive()` from the `NewCourse.vue` tried to handle it.

Left is PR and right is master. 
https://user-images.githubusercontent.com/54298311/106916504-419ff000-66d5-11eb-8f3d-c5b1e364d08e.mov